widgets/GCSBucketCreate-action.json

```diff
@@ -56,6 +56,19 @@
           }
         }
       ]
+    },
+    {
+      "label": "Auto Create",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Location",
+          "name": "location",
+          "widget-attributes": {
+            "placeholder": "Location of the gcs buckets"
```

Comment:
instead can we just show the default? If this can't be a selector right now?

Reply:
You just pass null to the client in order to create on default location, it is different for different gcp products. I think it is better to not specify them so that we can leave it to the clients to determine what the default is. And in case the client changes in the future, it will not break the plugin

Reply:
+1 to defaulting

Reply:
Changed to default